balances-adjustment/src/main/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationImpl.java

```diff
@@ -61,6 +62,14 @@ public BalanceComputationImpl(List<BalanceComputationArea> areas, ComputationMan
      */
     @Override
     public CompletableFuture<BalanceComputationResult> run(Network network, String workingStateId, BalanceComputationParameters parameters) {
+        return this.run(network, workingStateId, parameters, Reporter.NO_OP);
+    }
+
+    /**
+     * Run balances adjustment computation in several iterations
+     */
+    @Override
+    public CompletableFuture<BalanceComputationResult> run(Network network, String workingStateId, BalanceComputationParameters parameters, Reporter reporter) {
         Objects.requireNonNull(network);
         Objects.requireNonNull(workingStateId);
         Objects.requireNonNull(parameters);
```

Comment:
you can also do Objects.requireNonNull(reporter)

Reply:
done